build.sbt

```diff
@@ -112,11 +112,32 @@ lazy val http4s = project
   )
   .dependsOn(coreJVM)
 
+lazy val akkaHttp = project
```

Comment:
Can you also update the config.yml file in .circleci to compile this project? (same as the http4s one)

Reply:
Resolved in c1137ac